demos/http3/ossl-nghttp3.c

```diff
@@ -613,6 +614,12 @@ int OSSL_DEMO_H3_CONN_handle_events(OSSL_DEMO_H3_CONN *conn)
         if (ec == 0)
             break;
 
+        /*
+	 * we let SSL_write_ex2(3) to conclude the stream for us (send FIN)
```

Comment:
Minor nit: "let" doesn't need "to" here

```suggestion
	 * Let SSL_write_ex2(3) conclude the stream for us (send FIN)
```